config/jobs/kubernetes-sigs/image-builder/image-builder-ci.yaml

```diff
@@ -0,0 +1,30 @@
+postsubmits:
+  kubernetes-sigs/image-builder-konfigadm:
+    - name: post-image-builder-konfigadm
+      labels:
+        preset-dind-enabled: "true"
+        preset-image-builder-gcp-creds: "true"
```

Comment:
is those presets exist?

Reply:
I have removed the dind - but we need these presets created or are their existing presets that can be used that will give us the right creds?